WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -963,6 +963,7 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
                 let index = strongSelf.filterSettings.availablePostListFilters().indexOf(selectedValue as! PostListFilter) {
 
                 strongSelf.filterSettings.setCurrentFilterIndex(index)
+                strongSelf.syncItemsWithUserInteraction(false)
```

Comment:
I put it there to be as near its original location as possible, but I think you're right. Moved and pushed. Thanks 👍